File: groundhogDay.js
function groundhogDayPrediction(appearance) {
	if (typeof appearance !== 'boolean') {
		return 'No prediction this year.'
	} else if (appearance) {
		return "Looks like we'll have six more weeks of winter."
	}
	return "It's going to be an early spring."
}
//Demo test
console.log(groundhogDayPrediction('false')) //Output: No prediction this year.
console.log(groundhogDayPrediction(true)) //Output: Looks like we'll have six more weeks of winter.
console.log(groundhogDayPrediction(false)) //Output: It's going to be an early spring.

File: groundhogday.py
def groundhog_day_prediction(appearance):
    if type(appearance) is not bool:
        return "No prediction this year."
    elif appearance == True:
        return "Looks like we'll have six more weeks of winter."
    return "It's going to be an early spring."
#Demo test
print(groundhog_day_prediction("False"))#Output: No prediction this year.
print(groundhog_day_prediction(True))#Output: Looks like we'll have six more weeks of winter.
print(groundhog_day_prediction(False))#Output: It's going to be an early spring.
